lesson_15/tdd/tdd_app/src/test/java/com/codedifferently/lesson15/EmployeeManagerTest.java

```diff
@@ -0,0 +1,89 @@
+package com.codedifferently.lesson15;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class EmployeeManagerTest {
+  private EmployeeManager employeeManager;
+
+  @BeforeEach
```

Comment:
Credit to chatgpt here! I didn't know I would've needed a before each so every time i set up my tests , I can create a new manager to 'manage' the employees included